src/plugins/skills/mining/mining.ts

```diff
@@ -0,0 +1,91 @@
+import {ActionType, RunePlugin} from "@server/plugins/plugin";
```

Comment:
Please add spaces within import braces and use single ticks. `import { ActionType, RunePlugin } from '...';`